regolith/tools.py

```diff
@@ -755,7 +755,6 @@ def fragment_retrieval(coll, fields, fragment, case_sensitive = False):
     return ret_list
 
 def is_fully_appointed(appts, begin=None, end=None, now=None):
-    datearray = []
     status = [True, '']
     if begin and end and now:
```

Comment:
no need for now.  maybe make begin_date and end_date (please use these variable names for consistency) required args.

Reply:
done